### PR TITLE
Complete seeding my DB !-!-!

### PR DESCRIPTION
**Pull request by hannahmanfredi**
-^'-'^-

### CODE REVIEW

File: database/seed.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+    unavailableDays = function(array) {
+      let indexes = [];
+      let unavailable = [];
+      for (let i = 0; i <= 4; i++) {
+        let index = array[Math.floor(Math.random() * array.length)];
+        indexes.push[index];
+      }
+      indexes.forEach(index => {
+        unavailable.push(array[i]);
```
Comment (djian001) [inline database/seed.js:71]: I think you're always returning an empty array here. looks like you're trying to push array[i] outside of the for loop so i don't think it has access to "i"?

File: database/seed.js [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+      const twisselman = {
+        campId: 0,
+        price_per_night: 165,
+        guests: 2,
+        how_far_out: 6,
+        instant_book: true,
+        request_to_book: false,
+        year: 2021,
+        month: 6,
+        booked: [3, 10, 24],
```
Comment (djian001) [inline database/seed.js:134]: not 100% what booked represents but is it random individual dates that's been booked? or is it supposed to be multiple random days that are booked based on checkin and check out dates? Ex. Booked: May 3 - May 5, May 10 - May 15, May 21 - May22

File: database/seed.js [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+        guests: randomNumberMaker(),
+        how_far_out: far(),
+        instant_book: booleanMaker(),
+        request_to_book: !booleanMaker(),
+        year: 2021,
+        month: month,
+        booked: unavailable,
+        check_in_date: startDay(23, 1),
+        check_out_date: (startDay(23, 1) + randomNumberMaker()),
+        number_nights: randomNumberMaker(),
```
Comment (djian001) [inline database/seed.js:166]: Is number_nights supposed to represent number of days booked based on check_in_date and check_out_date? or just a random number? Do we care if it's truly being represented?

File: database/seed.js [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+        cleaning_fee: (price / 10),
+        weeknight_discount: discountMaker(),
+        addons_names: makeAddons(),
```
Comment (djian001) [inline database/seed.js:169]: I think addons_names contain only 1 string in the array of your returned data. Do we need to store this in an array?

**Review by djian001 — COMMENTED** [review]
Screen Shot 2021-05-09 at 1 52 03 PM

Here's a snapshot of the returned data i got back

File: database/seed.js (hunk 1 above)
Comment (hannahmanfredi) [reply database/seed.js:71]: Fixed, ty! Revised function which now creates an array of subarray date ranges here:

unavailableDays = function(array) {
  let indexes = [];
  let unavailable = [];
  let ranges = [1, 2, 3, 4, 5];
  let randomNumb = ranges[Math.floor(Math.random() * ranges.length)];
  while (indexes.length < randomNumb) {
    let index = array[Math.floor(Math.random() * array.length)];
    if (indexes.indexOf(index) === -1) {
      indexes.push(index);
    }
  }
  indexes.forEach(index => {
    let numb = array[index];
    let rangeEnd = (numb + randomNumb);
    let overflow = 0;
    let range = [];
    if (rangeEnd > array.length) {
      overflow += (rangeEnd - array.length);
      range = [array[index], overflow];
    } else {
      overflow += randomNumb;
      let end = (numb + overflow);
      range = [numb, end];
    }
    unavailable.push(range);
  });
  return unavailable;
};

File: database/seed.js (hunk 2 above)
Comment (hannahmanfredi) [reply database/seed.js:134]: Yes, booked should be range of dates that are now booked and will appear as unavailable on the calendar. They should be specified by checkin and check out dates on incoming GET request to Booking API

File: database/seed.js (hunk 3 above)
Comment (hannahmanfredi) [reply database/seed.js:166]: Fixed! Yes, we care! Now my number of nights =  (check_in_date - check_out_date)

File: database/seed.js (hunk 4 above)
Comment (hannahmanfredi) [reply database/seed.js:169]: No, rememberd it's Turbo's secondary service so we can get to this later if needed ty

**Pull request merged**